guides/common/modules/ref_supported-clients-in-registration.adoc

```diff
@@ -43,24 +54,12 @@ ifdef::satellite[]
 * s390x
 * ppc_64
 endif::[]
-
 ifdef::orcharhino[]
-[cols="2,3"]
-|===
-|Operating System |Architectures
-|Debian 10 |aarch64, ppc64le, amd64
-|Debian 11 |aarch64, amd64
-|Debian 12 |amd64
-|{EL} |x86_64
-|{SLES} 12 |amd64
-|{SLES} 15 |aarch64, ppc64le, amd64
-|Ubuntu 16.04 |amd64
-|Ubuntu 18.04 |amd64
-|Ubuntu 20.04 |aarch64, amd64
-|Ubuntu 22.04 |aarch64, amd64
-|===
+* x86_64/amd64
+* aarch64 and ppc64le for select operating systems
```

Comment:
```suggestion
* aarch64 and ppc64le for selected operating systems
```
?

Reply:
I think "select" is correct here: https://www.merriam-webster.com/dictionary/select

Maybe @bangelic can weigh in?

Reply:
It depends what you are trying to say. "Select" is an ambiguous word. OTOH, "Selected" implies that the user has selected these operating systems.

Maybe this:

- x86_64/amd64 is supported by all operating systems.
- aarch64 and ppc64le is supported by specific operating systems.

Reply:
Reworded.